Source/ACE/Command/Handlers/DebugCommands.cs

```diff
@@ -195,5 +195,29 @@ public static void CreatePortal(Session session, params string[] parameters)
         {
             LandblockManager.AddObject(PortalObjectFactory.CreatePortal(1234, session.Player.Position.InFrontOf(3.0f), "Test Portal", PortalType.Purple));
         }
+
+        /// <summary>
+        /// Debug command to saves the character from in-game.
+        /// </summary>
+        /// <remarks>Added a quick way to invoke the character save routine.</remarks>
+        /// <param name="session"></param>
+        /// <param name="parameters"></param>
+        [CommandHandler("save", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld)]
+        public static void Save(Session session, params string[] parameters)
+        {
+            session.SaveSession();
+        }
+
+        /// <summary>
+        /// Returns the Player's GUID
+        /// </summary>
+        /// <remarks>Added a quick way to access the player GUID.</remarks>
+        /// <param name="session"></param>
+        /// <param name="parameters"></param>
+        [CommandHandler("whoami", AccessLevel.Developer, CommandHandlerFlag.RequiresWorld)]
+        public static void WhoAmI(Session session, params string[] parameters)

```

Comment:
With multiple characters on screen/per account, this is helpful when comparing columns in the database.